apack.json

```diff
@@ -55,6 +55,7 @@
       "id": "ZAP",
       "category": "tools",
       "label": "ZCL Advanced Platform",
+      "icon": "icon.png",
```

Comment:
What about this file making it into the build?

Reply:
the workaround was that I was asked to put a string here so "icon" is defined